net/microsocks/files/microsocks.config

```
@@ -0,0 +1,11 @@
+config microsocks 'enabled'
+	option enabled '1'
+
+config microsocks 'config'
+	# If not set microsocks will use default iface
+	option bindaddr
+	option listenip 127.0.0.1
+	option port 1080
+	option user
+	option password
+	option auth_once 0 # Boolean, must be used together with user/pass
```

```suggestion

config microsocks 'config'
	option enabled '0'
	option bindaddr ''
	option listenip '127.0.0.1'
	option port '1080'
	option user ''
	option password ''
	option auth_once '0' # Boolean, must be used together with user/pass
```

Use single quotation marks to quote values, even empty.
Also, there's no need to put `enabled` option into a dedicated section.